fork-clone-workflow.md

```diff
@@ -33,9 +35,61 @@ own version of the https://github.com/mindfulness-at-the-computer/git-info reosi
 username is `joeuser`, the URL for your version of the repository will be
 https://github.com/joeuser/git-info.
 
-## Step 2 - Make a local clone of your fork
+## Setup Step 2 - Make a local clone of your fork
 This step is perfomed on the development system you will use to work on the source code 
-for the project. 
-### Work in progress (taking a snapshot of work on 2/27/18)
+for the project. You will [clone](git-glossary.md#clone) your fork on to your development machine. 
+Then you will configure your clone so that it knows how to find the main (organizational) repository
+from which the fork was created.
+
+* Change directory to the location on your development machine where you want to store the clone. You can clone
+to any location, but it is a good idea to keep your clones in a fixed location. In this example `joeuser` keeps
+cloned repositories in the `/opt/repos` directory.
+```bash
+cd /opt/repos
+```
+* Clone your fork using the `git clone` command. You can find the URL of your fork on the main [gihub](git-glossary.md#github)
+page of your fork. Click on the `Clone or download` button and a popup will show you the URL to use if you want to
+`Clone with SSH`. The SSH option is the easiest option to use but requires that you have confiured configured your
```

Comment:
I've always used the https option here, not sure which is more common but maybe we can put one of these in a footnote to make it easier to read. I'm not sure about this though, maybe it's better to leave it like it is now

Reply:
Do you mean that you work where you have to enter your github password whenever you access your fork of the organization repository? The only time I work that way is when I am on a machine that I don’t use often and don’t want to leave a an SSH key pair on the machine. 

If you find it acceptable to work where you enter your password each time, then perhaps we should document HTTPS as the default and added a link to a separate page that explainis how to use SSH access?